### PR TITLE
Allow to delete all backups

### PR DESCRIPTION
**Pull request by brunobar79**
Added an option to delete all backups with double confirmation.

PoW - single wallet: http://recordit.co/ZOyLbMTjig
PoW - multiple wallets: http://recordit.co/XqZLVwvzwz

### CODE REVIEW

**Comment by osdnk** [comment]
Since it's the single option, do we really need to have it inside the menu?

File: src/components/settings-menu/BackupSection/AlreadyBackedUpView.js [hunk 1]
```diff
@@ -145,6 +150,50 @@ export default function AlreadyBackedUpView() {
...
+              cancelButtonIndex: 1,
+              destructiveButtonIndex: 0,
+              message: `Are you sure you want to delete all iCloud Backups`,
+              options: ['YES, Delete All iCloud Backups', 'Cancel'],
+            },
+            async buttonIndex => {
+              if (buttonIndex === 0) {
+                const newWallets = { ...wallets };
+                Object.keys(newWallets).forEach(key => {
+                  delete newWallets[key].backedUp;
```
Comment (osdnk) [inline src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:177]: if you don't really need to, it's better to set `undefined`.
```suggestion
                 newWallets[key] = undefined;
```

Comment (osdnk) [reply src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:177]: https://levelup.gitconnected.com/5-facts-about-delete-operator-in-javascript-c16fd2588cd

it's generally a ton faster

**Review by osdnk — APPROVED** [review]

File: src/components/settings-menu/BackupSection/AlreadyBackedUpView.js (hunk 1 above)
Comment (osdnk) [reply src/components/settings-menu/BackupSection/AlreadyBackedUpView.js:177]: + few other places

**Comment by christianbaroni** [comment]
> Since it's the single option, do we really need to have it inside the menu?

@osdnk reason it’s hidden is bc I don’t think this option should be too heavily advertised

**Comment by brunobar79** [comment]
> > Since it's the single option, do we really need to have it inside the menu?
> 
> @osdnk reason it’s hidden is bc I don’t think this option should be too heavily advertised

LOL https://darkpatterns.org/types-of-dark-pattern/misdirection.html

**Review by skylarbarrera — APPROVED** [review]
looks good, works good 👌

**Pull request merged**